orc8r/gateway/python/magma/ctraced/BUILD.bazel

```diff
@@ -36,6 +36,6 @@ py_library(
         "rpc_servicer.py",
         "trace_manager.py",
     ],
-    visibility = ["//visibility:public"],
-    deps = [],
+    visibility = ["//visibility:private"],
```

Comment:
Is visibility private not be the default visibility setting, such that this line can be omitted completely?

Reply:
Fine for me either way! :+1: 
Not omitting is certainly more explicit. Not sure if an experienced Bazel user needs it though?

However that may be, do we want to define and document a policy, what we see best practice?
Maybe it is worthwhile to start a new section in the [Contribute Conventions Wiki](https://github.com/magma/magma/wiki/Contributing-Code-Conventions#languages) collecting all findings on that part?

Reply:
ui, did not know this one - will add some lines.